sync/awsdatabricks.py

```diff
@@ -280,10 +284,11 @@ def missing_message(input: str) -> str:
     else:
         reservations_response = Response(result={"Reservations": cluster_info["Reservations"]})
 
-    if not cluster_info or not cluster_info["Volumes"]:
+    if not cluster_info:
         volumes_response = Response(error=DatabricksError(message=missing_message("ebs volumes")))
     else:
-        volumes_response = Response(result={"Volumes": cluster_info["Volumes"]})
+        logger.warning(missing_message("ebs volumes"))
+        volumes_response = Response(result={"Volumes": cluster_info.get("Volumes", [])})
```

Comment:
It seems silly to return an error if there isn't ebs data. Instead just give a warning message.